demisto_sdk/commands/pre_commit/pre_commit_command.py

```diff
@@ -84,20 +87,21 @@ def _get_hooks(pre_commit_config: dict) -> dict:
         for repo in pre_commit_config["repos"]:
             for hook in repo["hooks"]:
                 hooks[hook["id"]] = hook
+                # if the hook has a skip key, we add it to the SKIPPED_HOOKS set
+                if hook.pop("skip", None):
```

Comment:
why `pop` over `get`?

Reply:
Because this is not a value that `pre-commit` framework reads. we want to remove it from the template after reading it.